code/game/objects/structures/billboard.dm

```diff
@@ -0,0 +1,35 @@
+/obj/structure/billboard
+	name = "blank billboard"
+	desc = "A blank billboard, with space for all kinds of advertising."
+	icon = 'icons/obj/billboard.dmi'
+	icon_state = "billboard_blank"
+	max_integrity = 1000
+	bound_width = 96
```

Comment:
bound_x, pixel_x and base_pixel_x to -32?

Reply:
what exactly did you mean by this? I assume you're meaning to give it a default offset?

Reply:
yeah